### PR TITLE
htaccess comments

### PR DESCRIPTION
**Pull request by brianteeman**
Simple PR to improve the comments and fix some grammar

### CODE REVIEW

File: htaccess.txt [hunk 1]
```diff
@@ -27,16 +28,17 @@ Options -Indexes
 RewriteEngine On
 
 ## Begin - Rewrite rules to block out some common exploits.
-# If you experience problems on your site block out the operations listed below
-# This attempts to block the most common type of exploit `attempts` to Joomla!
+# If you experience problems on your site then coment out the operations listed below
```
Comment (Twincarb) [inline htaccess.txt]: @brianteeman comment not coment

File: htaccess.txt [hunk 2]
```diff
@@ -52,7 +54,7 @@ RewriteRule .* index.php [F]
 ## End - Custom redirects
 
 ##
-# Uncomment following line if your webserver's URL
+# Uncomment the following line if your webserver's URL
```
Comment (Twincarb) [inline htaccess.txt]: @brianteeman is it webservers or web servers google shows it as 2 words, either way it shouldn't have an ' in it

File: htaccess.txt (hunk 1 above)
Comment (brianteeman) [inline htaccess.txt]: Doh

File: htaccess.txt (hunk 2 above)
Comment (brianteeman) [inline htaccess.txt]: I prefer one word just as website is one word.

Are you sure it should not have the apostrophe? It is a possessive "the url
OF the webserver

Comment (Twincarb) [inline htaccess.txt]: my bad, it's correct!

Comment (brianteeman) [inline htaccess.txt]: no problem I really appreciate the check

**Comment by Twincarb** [comment]
I have tested this item :white_check_mark: successfully on f25818ce35ada7ac694036340ea1e674eff26936<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11062">issues.joomla.org/joomla-cms/11062</a>.</sub>

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on f25818ce35ada7ac694036340ea1e674eff26936<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11062">issues.joomla.org/joomla-cms/11062</a>.</sub>

**Comment by brianteeman** [comment]
RTC - 

@gunjanpatel you have to make a comment for the bot to work and set it to RTC on github<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11062">issues.joomla.org/joomla-cms/11062</a>.</sub>

**Comment by gunjanpatel** [comment]
ohh ok getting you. Bot will be trigger only if I write RTC in comment. Thanks :)

**Comment by brianteeman** [comment]
Actually the bot is triggered if you write anything

On 11 July 2016 at 10:28, Gunjan Patel <notifications@github.com> wrote:

> ohh ok getting you. Bot will be trigger only if I write RTC in comment.
> Thanks :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/11062#issuecomment-231685036>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABPH8eYATtLC5wpzS2F3ovkafeHzos2Bks5qUgy4gaJpZM4JIz3r>
> .
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Pull request merged**